### DISCUSSION TITLE
Problema al comprobar la instalación de homebrew

### DISCUSSION THREAD

**Discussion by dectun** [Homebrew/discussions#3170] [Getting started]
Hola. Intento instalar MVE, y después de la instalación correcta de hombrew, no puedo usar los comandos para brew doctor:  
(https://github.com/simonfuhrmann/mve/wiki/Build-Instructions-for-OS-X)

El mensaje del terminal dice;
 zhs: comando no encontrado: brew

Lo hago sobre un macbook air M1.
gracias

**Comment by carlocab** [comment]
Did you follow the `Next Steps` shown to you by the installer? It should've looked something like
```
- Add Homebrew to your PATH in ~/.zprofile by running these two commands:
    echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> ~/.zprofile
    eval $(/opt/homebrew/bin/brew shellenv)
```

**Reply by dectun** [reply]
That`s good. It`s running. Your a god. 

very much thanks

**Reply by dectun** [reply]
ok. Tengo otro problema. Al llegar al paso para el comando;

cd mve; make

me dice;

clang: error: the clang compile does not support `-march=native`
make[2]: *** [arguments.o] Error 1
make[1]: *** [all] Erro 2
make: *** [all] Error 2

No sé que he hecho mal. Pero la clonación del repositorio MVE fue correcta. Aunque la realicé antes de este paso que me has arreglado. ¿puede ser eso?

thx

**Reply by carlocab** [reply]
That looks like an `mve` problem and not `brew` problem. You'll need to follow this up at https://github.com/simonfuhrmann/mve.

**Reply by mendezgutierrezh** [reply]
Very much thanks

**Comment by UliRod** [comment]
Cuando intento correr brew en un terminal nuevo me dice zsh: command not found: brew

despues de installar: 
echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> ~/.zprofile
eval $(/opt/homebrew/bin/brew shellenv)
puedo ejecutar brew en esa misma ventana sin problema

pero cuando abro una ventana nueva y escribo -brew- 
me sale de nuevo el error de 
zsh: command not found: brew

que pudiera estar pasando?  :(

**Reply by osalbahr** [reply]
What do `echo $0` and `cat ~/.zprofile` tell you?

**Reply by UliRod** [reply]
Me di cuenta que en mi terminal al abrirla tenía un error como este
.zshrc:export:1: not valid in this context

después de buscar ejecute esto 
Use a text editor like nano to open ~/.zshrc

/usr/bin/nano ~/.zshrc
Delete the line with ctrl-K if you don't need it.
Save with ctrl-X.
Restart Terminal.

Y ahora ya puedo usar el comando brew en la terminal 
Muchas gracias por la ayuda
